cmd/kubeadm/app/cmd/init.go

```diff
@@ -115,41 +127,58 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 	return cmd
 }
 
-// RunInit executes master node provisioning, including certificates, needed static pod manifests, etc.
-func RunInit(out io.Writer, cmd *cobra.Command, args []string, cfg *kubeadmapi.MasterConfiguration) error {
+type Init struct {
+	cfg *kubeadmapi.MasterConfiguration
+}
+
+func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration) (*Init, error) {
+	if cfgPath != "" {
+		b, err := ioutil.ReadFile(cfgPath)
+		if err != nil {
+			return nil, fmt.Errorf("unable to read config from %q: %v", cfgPath, err)
```

Comment:
We've had `[%v]` in other places, which sort of implies _"you can read this, but it might not be very meaningful to you"_. I know that `: %v` is a wider convention, I'd be happy to switch if you insist, but I'm not sure it's actually a great convention... Happy to discuss in another thread.